source/containers.tex

```diff
@@ -17053,16 +17053,14 @@
 
 \pnum
 The effect of calling a constructor
-that takes
-both \tcode{key_container_type} and \tcode{mapped_container_type} arguments with
-containers of different sizes is undefined.
+that takes both \tcode{key_container_type} and \tcode{mapped_container_type} arguments
+with containers of different sizes
+is undefined.
 
 \pnum
 The effect of calling a constructor or member function
-that takes a \tcode{sorted_unique_t} argument with
-a container, containers, or range
-that is not sorted with respect to \tcode{key_comp()}, or
-that contains equal elements,
+that takes a \tcode{sorted_unique_t} argument
+with a container, containers, or range that is not sorted with respect to \tcode{key_comp()}, or that contains equal elements,
```

Comment:
I believe a "container" _is_ a "range," so should this (and throughout) just say "with a range or ranges that (is/are)...", "with a range or containers that (is/are)...", or any variation along those lines? I wasn't able to come up with a phrasing that wasn't icky. IMO the benefit of not-too-awful grammar is worth the cost of redundancy here.